src/components/puck/atoms/button.tsx

```diff
@@ -19,6 +18,22 @@ const buttonVariants = cva(
         large: "h-11 px-8",
         icon: "h-10 w-10",
       },
+      fontSize: {
```

Comment:
I don't think these need to have the ! (but double check that). We added the ! to heading because the fontSize variants can conflict with the headingLevel text size variants

Reply:
They do need it, the font size on the button doesn't seem to change without it.

Reply:
ah there's  `text-button-fontSize` in the base string at the top. Try removing that (and update the default below to use text-button-fontSize

Reply:
Ah yeah that worked.